tests/ideas/rules/test_rules_add.py

```diff
@@ -0,0 +1,93 @@
+from datetime import timedelta
+
+import pytest
+import rules
+from freezegun import freeze_time
+
+from apps.ideas import phases
+from tests.helpers import setup_phase
+from tests.helpers import setup_users
+
+perm_name = 'meinberlin_ideas.add_idea'
+
+
+def test_perm_exists():
+    assert rules.perm_exists(perm_name)
+
+
+@pytest.mark.django_db
+def test_rules_pre_phase(phase_factory, user):
+    phase, module, project, _ = setup_phase(phase_factory, None,
+                                            phases.CollectPhase)
+    anonymous, moderator, initiator = setup_users(project)
+
+    assert project.is_public
+    with freeze_time(phase.start_date - timedelta(days=1)):
```

Comment:
Would be nice to define helper for all the freeze functions. E.g. freeze_phase/pre_phase/after_phase.